src/main/java/de/ait/secondlife/services/utilities/EntityUtilities.java

```diff
@@ -1,19 +1,34 @@
 package de.ait.secondlife.services.utilities;
 
 import de.ait.secondlife.constants.EntityTypeWithImages;
+import de.ait.secondlife.constants.ImageConstants;
+import de.ait.secondlife.domain.entity.ImageEntity;
+import de.ait.secondlife.domain.entity.User;
+import de.ait.secondlife.exception_handling.exceptions.NoRightToChangeException;
+import de.ait.secondlife.exception_handling.exceptions.bad_request_exception.PathWrongException;
 import de.ait.secondlife.exception_handling.exceptions.not_found_exception.BadEntityTypeException;
+import de.ait.secondlife.security.Role;
 import de.ait.secondlife.services.interfaces.*;
 import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
+import javax.security.auth.login.CredentialException;
+import java.util.HashSet;
+import java.util.Set;
+
 
 @Component
 @RequiredArgsConstructor
-public class EntityUtilities {
+public class EntityUtilities implements ImageConstants {
```

Comment:
IMHO, the solution with this utility class and its methods is not very obvious. I'd like to discuss

Reply:
fixed